cg/src/utils/blockchain/contract-bytecode.js

```diff
@@ -1,4 +1,4 @@
 const contractByteCode =
```

Comment:
Where did you get this compiled code?
When u change the bytecode, you also have to change the ABI (Application Binary Interface)

Reply:
I used https://remix.ethereum.org to compile contract. Actually, we need to make compilation to be a part of contract deploy process. I am not sure that keep compiled ABI and bytecode in the repo is a good idea either.

Reply:
Alright, we're good then